www/hodl01/semestralka/results.html

```diff
@@ -0,0 +1,323 @@
+    <link rel="stylesheet" href="style.css" media="screen">
+    <link href="https://fonts.googleapis.com/css?family=Rajdhani" rel="stylesheet">
+    <link rel="shortcut icon" href="IMG/favicon.ico" type="image/x-icon">
+    <title>Výsledky | Lukáš Hodboď</title>
+</head>
+
+<body>
+    <header id="header" class="pageHeader">
+        <div class="bigsocialicons">
+            <p><a href="https://www.facebook.com/lukas.hodbod" target="_blank"><img src="IMG/facebook.png" alt="facebook"></a>&nbsp;<a
+                    href="https://www.youtube.com/user/hodbod59/" target="_blank"><img src="IMG/youtube.png" alt="youtube"></a>&nbsp;<a
+                    href="https://www.instagram.com/lukashodbod/" target="_blank"><img src="IMG/insta_ikona.png" alt="insta ikona"
+                        width="94" height="86"></a></p>
+        </div>
+        <h1 class="name"><a class="name" href="index.html"><span class="firstName">Lukáš</span><br>Hodboď</a></h1>
+
+    </header>
+    <nav id="nav">
+        <ul class="nav">
+            <li><a href="index.html">O mně</a></li>
+            <li><a class="active" href="results.html">Výsledky</a></li>
+            <li><a href="foto.html">Galerie</a></li>
+            <li><a href="media.html">Média</a></li>
+            <li><a href="makacenko.html">Tréninková skupina</a></li>
+            <li><a href="contact.html">Kontakty</a></li>
+        </ul>
+    </nav>
+
+    <main class="maxWidth">
+        <div class="blackBox">
+            <h2>Výsledky závodu 2018</h2>
+            <table class="dataTable">
+                <thead>
+                    <tr>
+                        <th class="left">Datum</th>
+                        <th class="left">Soutěž (pořadatel)</th>
+                        <th class="left">Disciplína</th>
+                        <th>Poř.</th>
+                        <th class="left">Místo</th>
+                        <th class="right">Výkon</th>
+                    </tr>
+                </thead>
+                <tbody>
+                    <tr>
+                        <td>01.09.2018</td>
+                        <td>Mistrovství ČR mužů a žen do 22 let na dráze (SSK Vítkovice, z.s.)</td>
+                        <td><a href="https://online.atletika.cz/vysledky/31961/110#discid2137915">800 m/U23M/R1 </a></td>
+                        <td class="center">
+                            1
+
+                        </td>
+                        <td>Ostrava</td>
+                        <td class="right bold">1:55.73</td>
+                    </tr>
+                    <tr>
+                        <td>01.09.2018</td>
+                        <td>Mistrovství ČR mužů a žen do 22 let na dráze (SSK Vítkovice, z.s.)</td>
+                        <td><a href="https://online.atletika.cz/vysledky/31961/110#discid2137918">800 m/U23M/F </a></td>
+                        <td class="center">
+                            <img class="positionimg" src="/IMG/240px-Gold_medal_icon.png" alt="1.">
```

Comment:
vetsina obrazku nenalezena, je to kvuli tomu lomitku na zacatku, pokud je slozka img ve stejne slozce jako tento soubor, lomitko tam nema byt